packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss

```diff
@@ -5,6 +5,7 @@
 
   > atomic-icon {
     width: 100%;
+    color: white;
```

Comment:
Why was this change needed?

Reply:
The icons in our package (from JSUI) all had `fill="#fff"`, the ones from vapor don't have that attribute, so they've been replace by this property